src/main/java/net/fabricmc/loom/configuration/providers/minecraft/MinecraftProvider.java

```diff
@@ -185,6 +186,10 @@ public MinecraftVersionMeta getVersionInfo() {
 		return Objects.requireNonNull(metadataProvider, "Metadata provider not setup").getVersionMeta();
 	}
 
+	public boolean canMergeJars() {
```

Comment:
Im not too keen on this name, it should be more about the version, and not if it can be merged or not. Right now with this the the split or single providers will return true for `canMergeJars`, this isnt very clear. I think something along the lines of `isLegacyVersion` may be better?

Reply:
`isLegacyVersion` or `isLegacyMinecraft` sounds good, perhaps `isPre1_3` though that doesn't leave any room for it to change I suppose.